android-plugin/src/main/java/com/zendesk/unity/ZDK_Plugin.java

```diff
@@ -231,7 +239,12 @@ public void viewArticle(final String id){
         provider.getArticle(idLong, new ZendeskCallback<Article>() {
             @Override
             public void onSuccess(Article article) {
-                ViewArticleActivity.startActivity(getActivity(), article);
+                ViewArticleActivity.startActivity(
```

Comment:
In iOS voting can be controlled via the instance singleton but because the Android version works differently I keep the value myself and apply it on next Activity display. We don't need this for the HelpCenter because it receives it as a parameter.

I preferred this approach instead of adding a new parameter to limit changes as much as possible and to make the currently existing `ZDKConfig.SetArticleVotingEnabled` c# method available for both platforms.